src/main/java/frc/trigon/robot/motorsimulation/SimpleMotorSimulation.java

```diff
@@ -0,0 +1,40 @@
+package frc.trigon.robot.motorsimulation;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+
+public class SimpleMotorSimulation extends MotorSimulation {
+    private final DCMotorSim motorSimulation;
+
+    public SimpleMotorSimulation(DCMotor motor, double gearRatio, double momentOfInertia) {
+        motorSimulation = new DCMotorSim(motor, gearRatio, momentOfInertia);
+    }
+
+    @Override
+    double calculateFeedforward(double ks, double kg, double kv, double ka) {
+        SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(ks, kv, ka);
+        return feedforward.calculate(getVelocityRevolutionsPerSecond());
+    }
```

Comment:
1. I don't want you to create the feedforward class every time. Instead, do the calculation yourself.
2. Since feedforward is open-loop is accepts only the target position / vel. 
3. Make sure you're comparing in the unit of the conversion factor.